app/src/components/AppConnection/ServiceDataForm/ServiceForm.js

```diff
@@ -54,13 +54,34 @@ class ServiceForm extends Component {
             <TextInput name={v.name} onChange={this.handleChange} placeholder={v?.placeholder} required={v.required} wide />
           </Field>
         ))}
+
+        {service.description ? (
+          <div
+            css={`
+              margin-top: 5px;
+            `}
+          >
+            <Info>
+              {service?.description?.text}{' '}
+              {service?.description?.link ? (
+                <a href={service?.description?.link} target='_blank' rel='noopener noreferrer'>
+                  example
```

Comment:
```suggestion
                  (example)
```
---
Note: It's not nice to hardcode the text of the link here

Reply:
Fix on https://github.com/mesg-foundation/aragon/pull/25